src/Dungeon/DungeonGenerator.elm

```diff
@@ -323,10 +323,10 @@ findAndUpdateRoom targetRoom ({ rooms, activePoints } as model) =
                 someOtherPoint ->
                     someOtherPoint :: updatedPoints
 
-        activePoints' =
+        activePoints_ =
```

Comment:
Exactly. This is older code, i've been moving towards semantically naming transformed state and it makes the code easier for me to understand.